.travis.yml

```diff
@@ -20,6 +20,15 @@ matrix:
           - libc6-dev-i386-cross
           - cmake
           - cmake-data
+    - env: C_COMPILER=i686-linux-gnu-gcc-8 CFLAGS=-march=i686 -DSIMDE_NO_SHUFFLE_VECTOR -m-no-avx2 -m-no-avx -m-no-sse4.2 -m-no-sse4.1 -m-no-ssse3 -m-no-sse3 -m-no-sse2 -m-no-sse
```

Comment:
You can use `- env:` with hash value like this. And `CFLAGS` might need the single quote `'` or double quote `*`.

```
- env:
  - C_COMPILER=i686-linux-gnu-gcc-8
  - CFLAGS='-march=i686 -DSIMDE_NO_SHUFFLE_VECTOR -m-no-avx2 -m-no-avx -m-no-sse4.2 -m-no-sse4.1 -m-no-ssse3 -m-no-sse3 -m-no-sse2 -m-no-sse'
```

Reply:
@junaruga Good catch, thanks!